infrastructure/docker-registry/README.md

```diff
@@ -1,74 +1,72 @@
 - leverage the high-speed network that connects your servers, avoiding to consume precious Internet bandwidth to transfer images stored in the Docker Hub public service.
+- [Proxy Cache](https://goharbor.io/docs/2.4.0/administration/configure-proxy-cache/) to not exceed Docker Hub’s rate limit policy.
+- have a [vulnerability scanner](https://goharbor.io/docs/2.4.0/administration/vulnerability-scanning/)
+- manage your [Chart](https://goharbor.io/docs/edge/working-with-projects/working-with-images/managing-helm-charts/)
+- [Create System Robot Accounts](https://goharbor.io/docs/2.4.0/administration/robot-accounts/)
 
 Finally, consider that, in this Kubernetes setup, users instantiate mainly VMs, whose image may be rather large. Allowing users to download the VM image locally, instead of from a remote server, would greatly impact on their quality of experience in term of time required to start their service.
 
 ## Docker Registry Helm Chart
-We used the [Docker Registry Helm Chart](https://github.com/helm/charts/tree/master/stable/docker-registry), that is a Kubernetes chart to deploy a private Docker Registry where we have appropriately modified the values in [values.yaml](https://github.com/helm/charts/blob/master/stable/docker-registry/values.yaml) file.
+We used the [Harbor Registry Helm Chart](https://github.com/goharbor/harbor-helm), that is a Kubernetes chart to deploy a private Harbor Registry where we have appropriately modified the values in [values.yaml](https://github.com/goharbor/harbor-helm/blob/master/values.yaml) file.
+
+### Pre-requisites
+
+  1. [Kubernetes cluster 1.10+](https://kubernetes.io/releases/download/)
+  2. [Helm 2.8.0+](https://helm.sh/docs/intro/install/)
+  3. [High available ingress controller](https://github.com/netgroup-polito/CrownLabs/tree/master/infrastructure/ingress-controller) (Harbor does not manage the external endpoint)
+  4. [High available PostgreSQL 9.6+](#https://github.com/netgroup-polito/CrownLabs/tree/master/infrastructure/identity-provider#postgresql-operator) (Harbor does not handle the deployment of HA of database)
+  5. [High available Redis](#https://github.com/bitnami/charts/tree/master/bitnami/redis/#installing-the-chart) (Harbor does not handle the deployment of HA of Redis)
+  6. PVC that can be shared across nodes or external object storage
+
 
 ### Configuration
-These are our modification of the various file.
-
-  1. Annotations concerning the authentication in the Ingress controller.
-```yaml
-annotations:
-   nginx.ingress.kubernetes.io/auth-realm: Authentication Required - ok
-   nginx.ingress.kubernetes.io/auth-secret: basic-auth
-   nginx.ingress.kubernetes.io/auth-type: basic
-```
+These are our modification of the various file. They all refer to the values.yaml file except the configuration n. 5 and 6. These 2 configuration refer respectively to postgres postgres_cluster_manifest.yaml and redis_service_values.yaml file.
```

Comment:
```suggestion
The following outlines the most relevant modifications applied to the Harbor `values.yaml` file:
```

In the list below, I would just outline the main modifications, to avoid them from becoming stale as soon as something is updated.